README.md

```diff
@@ -26,6 +26,13 @@ template databases for lightning-fast test execution.
 go get github.com/andrei-polukhin/pgdbtemplate
 ```
 
+## Choose a PostgreSQL Driver
+
+Choose either one of these PostgreSQL drivers:
+
+- `github.com/andrei-polukhin/pgdbtemplate-pq` (for `database/sql` with `lib/pq`)
+- `github.com/andrei-polukhin/pgdbtemplate-pgx` (for `pgx/v5` with connection pooling)
```

Comment:
nit: AFAIK this is the more popular option so you might want to list it first.

Reply:
Sound thinking, done!